spec/requests/main_page_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.feature 'MainPage', type: :feature do
+  it 'should display the root path' do
+    visit root_path
+    expect(page).to have_content('List of all users')
```

Comment:
Also kindly test whether the correct template is being rendered by using eg., `expect(response).to render_template(:show)`